content/docs/user-guide/interactivity/robotics/importing-robot.md

```diff
@@ -0,0 +1,115 @@
+- Select (using the checkbox) if the importer should copy meshes into the O3DE project assets folder or not
+- Click **Next**
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_load_file.png)
+
+> Note: if you leave the **Import meshes...** checkbox unchecked, the importer will try to find matching meshes in the project asset library. Verification of identity of meshes is done by CRC comparison.
+
+5. If you are importing a XACRO file that features parameters to be set, you will be asked to do so on the next page. Parameters can be edited by double-clicking on the value. Click **Next** when finished.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_XACRO_parameters.png)
+
+> Note: which parameters must be set for succesful import strictly depends on the XACRO project you are importing. Please refer to its documentation for more information.
+
+If import fails you will see a message with the output of the ROS 2 XACRO executable.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_fail.png)
+
+### Processing robot assets
+
+6. If the XACRO or URDF file is loaded correctly you will see a table with a list of all mesh files (assets) used in the model. Each row of the table describes one asset, with information divided into the following columns:
+
+- **URDF mesh path** - path extracted from the URDF
+- **Resolved mesh from URDF** - file path of the mesh located in user's filesystem
+- **Product asset** - location of the generated asset (`*.azmodel` or `*.pxmesh`) in the O3DE project assets directory
+- **Source asset** - location of the asset mesh file in the O3DE project assets directory
+- **Type** - the type of the asset, which can be `visual` or `collider`
+
+Besides this information, there is an indication if the asset was imported correctly (green) or if the import failed (red).
+
+Additional information about each asset can be obtained by double-clicking its row. This will open the O3DE Asset Processor showing import details. This information may be especially helpful in case of assets that were not imported correctly.
+
+Click **Next** when finished.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_mesh_list.png)
+
+**What to do if there are problems with importing assets?**
+If the Source asset field is marked as failed, it is caused by a URDF importer failure to find a reference to the mesh file in the file system. In such a situation, double-check whether the robot's description package was built and sourced correctly.
+If the Product asset field is marked as failed, it is caused by the Asset Builder failing to build the product asset. Refer to the Asset Processor output to resolve the issue.
+You can do either:
+
+- click **Back**, resolve problems with mesh files and click **Next** to retry import, or
+- proceed by clicking **Next** and finish URDF import. Missing meshes can be added afterward in the O3DE editor.
```

Comment:
Add newline to create separate paragraph
Clarify the instructions you're providing and what they do. 
```suggestion

You can either resolve the issues now or after you finish using Robot Importer: 
- To resolve, click **Back** and resolve problems with the mesh files. Then click **Next** to retry import.
- To resolve later, click **Next** and finish URDF import. Then add the missing meshes in the O3DE Editor.
```